Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM oven/bun:slim
+
+WORKDIR /app
+
+COPY package*.json ./
```

Comment:
Bun's lockfile isn't `package-lock.json`, so you cannot use glob pattern here.
Instead you have to specify it directly (`bun.lock`)